 
 This patch should fix a problem with th %l editing in the 
 client (FD) where it edited nothing. With this fix, it should
 edit "since".  
 Apply the patch to 1.36.2 with:

 cd <bacula-source>
 patch -p0 <1.36.2-level.patch
 make
 ...

Index: src/version.h
===================================================================
RCS file: /cvsroot/bacula/bacula/src/version.h,v
retrieving revision 1.403.2.5
retrieving revision 1.403.2.5.2.1
diff -u -r1.403.2.5 -r1.403.2.5.2.1
--- src/version.h	27 Feb 2005 21:54:02 -0000	1.403.2.5
+++ src/version.h	4 Mar 2005 09:14:15 -0000	1.403.2.5.2.1
@@ -1,8 +1,8 @@
 /* */
 #undef  VERSION
 #define VERSION "1.36.2"
-#define BDATE   "28 February 2005"
-#define LSMDATE "28Feb05"
+#define BDATE   "04 March 2005"
+#define LSMDATE "04Mar05"
 
 /* Debug flags */
 #undef  DEBUG
Index: src/filed/job.c
===================================================================
RCS file: /cvsroot/bacula/bacula/src/filed/job.c,v
retrieving revision 1.91.4.2
retrieving revision 1.91.4.2.2.1
diff -u -r1.91.4.2 -r1.91.4.2.2.1
--- src/filed/job.c	25 Feb 2005 09:47:06 -0000	1.91.4.2
+++ src/filed/job.c	4 Mar 2005 09:14:16 -0000	1.91.4.2.2.1
@@ -3,7 +3,7 @@
  *
  *    Kern Sibbald, October MM
  *
- *   Version $Id$
+ *   Version $Id$
  *
  */
 /*
@@ -1139,7 +1139,7 @@
       buf = get_memory(dir->msglen+1);
       utime_t since_time, adj;
       btime_t his_time, bt_start, rt=0, bt_adj=0;
-      if (jcr->JobLevel == 0) {
+      if (jcr->JobLevel == L_NONE) {
 	 jcr->JobLevel = L_SINCE;     /* if no other job level set, do it now */
       }
       if (sscanf(dir->msg, "level = since_utime %s mtime_only=%d",
